src/main/scala/org/allenai/plugins/DockerBuildPlugin.scala

```diff
@@ -171,6 +183,15 @@ object DockerBuildPlugin extends AutoPlugin {
   // Common settings / tasks / utilities used across tasks.
   //////////////////////////////////////////////////////////////////////////////////////////////////
 
+  def hashDirectory(imageDir: File): String = {
+    val allFiles = PathFinder(imageDir).***.filter(_.isFile).get
+    Utilities.hashFiles(allFiles, imageDir)
+  }
+
+  def imageNameWithLabel(imageDir: File, imageName: String): String = {
```

Comment:
Removed the directory hashing so it has to be called explicitly each time. This reduces the function to a simple string concatenation, but I think we should keep it for expressiveness.